src/content/en/tools/chrome-devtools/security/index.markdown

```diff
@@ -0,0 +1,93 @@
+---
+layout: shared/narrow
+title: "Security"
+description: "Use the Security Panel to ensure that all resources on your 
+site are protected with HTTPS."
+published_on: 2015-12-22
+updated_on: 2015-12-22
+order: 2
+authors:
+  - kaycebasques
+translation_priority: 0
+key-takeaways:
+  security:
+    - Use the Security Overview to instantly find out whether the current 
+      page is secure or non-secure.
+    - Inspect individual origins to view connection and certificate details
+      (for secure origins) or to find out exactly which requests are
+      unprotected (for non-secure origins).
+---
+
+HTTPS provides [critical security and data integrity][why-https] 
+both for your websites and for the people that entrust your websites 
+with their personal information. Use the Security Panel in Chrome DevTools 
+to debug security issues and ensure that you have properly implemented 
+HTTPS on your websites.
+
+{% include shared/toc.liquid %}
+
+{% include shared/takeaway.liquid list=page.key-takeaways.security %}
+
+## Security Overview
+
+To view the overall security of a page, open DevTools and go to the 
+Security Panel. 
+
+The first thing you see is the Security Overview. At a glance, the 
+Security Overview tells you whether the page is secure. A secure page is 
+indicated with the message `This page is secure (valid HTTPS).`
+
+![security overview, secure page](/web/tools/chrome-devtools/security/images/overview-secure.png)
+
+Click **View certificate** to view the server certificate for the 
+[main origin][same-origin-policy]. 
+
+![view certificate](/web/tools/chrome-devtools/security/images/view-certificate.png)
+
+A non-secure page is indicated with the message `This page is not secure.`
+
+The Security Panel distinguishes between two types of non-secure pages. 
+If the requested page is served over HTTP, then the main origin is flagged as 
+not secure. 
+
+![security overview, non-secure main origin](/web/tools/chrome-devtools/security/images/overview-non-secure.png)
+
+If the requested page is retrieved over HTTPS, but the page then goes on 
+to retrieve content from other origins using HTTP, then the page is still 
```

Comment:
> content from other origins

Should read "content from any origin". Since, even if you serve the page over HTTPS, if you serve any resources from the same origin over HTTP (say by a hard-coded URL and it doesn't redirect) then it would still trigger mixed-content.